cmd/standalone/main.go

```diff
@@ -55,22 +60,32 @@ func main() {
 	metricsFactory := xkit.Wrap("jaeger-standalone", expvar.NewFactory(10))
 	memStore := memory.NewStore()
 
-	builder := agentApp.NewBuilder()
-	builder.Bind(flag.CommandLine)
 	flag.Parse()
```

Comment:
unfortunately, it is. There are still some global flags which cannot be converted to viper/cobra because they are used in other commands. There will be a follow-up PR to handle it.